config/browser-extension/ext.base.config.js

```diff
@@ -22,6 +22,9 @@ const manifest = merge(manifestSettings.base, {
     default_icon: 'assets/icon.png',
     default_popup: 'popup.html',
   },
+  icons: {
+    '16': 'assets/icon.png',
```

Comment:
why '16'? 😛

Reply:
That's the one used context menu display. Could have left a comment..